Models/Management/Service.cs

```diff
@@ -13,13 +13,13 @@ public class Service : IService
 
         public struct Names
         {
-            public const short MonitorService = 1;
-            public const short EndpointTracker = 2;
-            public const short CSTTracker = 3;
-            public const short NancyGateway = 4;
-            public const short PostOffice = 5;
-            public const short WebConfBackup = 6;
-            public const short SiteTransactions = 7;
+            public static short MonitorService => 1;
+            public static short EndpointTracker => 2;
+            public static short CSTTracker => 3;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename property 'CSTTracker' to match pascal case naming rules, consider using 'CstTracker'.](https://app.codacy.com/app/onurakkaya/DSM.Core/pullRequest?prid=4732028)